extension/content/opt-out.css

```diff
@@ -1,5 +1,11 @@
+:root {
+    --color: rgb(255, 255, 255);
```

Comment:
Added CSS variable so it could handle change depending on the twitter theme change.